<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1136</width>
    <height>923</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>微軟正黑體</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>EyeAnalysis</string>
  </property>
  <property name="windowIcon">
   <iconset resource="icon.qrc">
    <normaloff>:/deepicon.png</normaloff>:/deepicon.png</iconset>
  </property>
  <property name="iconSize">
   <size>
    <width>453</width>
    <height>106</height>
   </size>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout_6">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <layout class="QVBoxLayout" name="verticalLayout">
            <item>
             <widget class="QLabel" name="label">
              <property name="minimumSize">
               <size>
                <width>632</width>
                <height>30</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>632</width>
                <height>30</height>
               </size>
              </property>
              <property name="font">
               <font>
                <family>微軟正黑體</family>
                <pointsize>10</pointsize>
                <weight>75</weight>
                <bold>true</bold>
               </font>
              </property>
              <property name="text">
               <string>攝影機選單</string>
              </property>
             </widget>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_5">
              <item>
               <widget class="QComboBox" name="comboBox_camList">
                <property name="enabled">
                 <bool>true</bool>
                </property>
                <property name="minimumSize">
                 <size>
                  <width>0</width>
                  <height>30</height>
                 </size>
                </property>
                <property name="font">
                 <font>
                  <family>Arial</family>
                  <pointsize>10</pointsize>
                  <weight>75</weight>
                  <bold>true</bold>
                 </font>
                </property>
                <property name="editable">
                 <bool>false</bool>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="btnRefresh">
                <property name="minimumSize">
                 <size>
                  <width>90</width>
                  <height>30</height>
                 </size>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>90</width>
                  <height>30</height>
                 </size>
                </property>
                <property name="font">
                 <font>
                  <family>Arial Black</family>
                  <pointsize>10</pointsize>
                  <weight>75</weight>
                  <bold>true</bold>
                 </font>
                </property>
                <property name="text">
                 <string>refresh</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout">
              <item>
               <widget class="QLineEdit" name="lineEdit_camName">
                <property name="enabled">
                 <bool>true</bool>
                </property>
                <property name="minimumSize">
                 <size>
                  <width>0</width>
                  <height>30</height>
                 </size>
                </property>
                <property name="font">
                 <font>
                  <family>Arial</family>
                  <pointsize>8</pointsize>
                 </font>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="btnEditCamName">
                <property name="minimumSize">
                 <size>
                  <width>90</width>
                  <height>30</height>
                 </size>
                </property>
                <property name="font">
                 <font>
                  <family>微軟正黑體</family>
                  <pointsize>10</pointsize>
                  <weight>75</weight>
                  <bold>true</bold>
                 </font>
                </property>
                <property name="text">
                 <string>修改相機名</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QLabel" name="right_label">
            <property name="minimumSize">
             <size>
              <width>640</width>
              <height>480</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>640</width>
              <height>480</height>
             </size>
            </property>
            <property name="baseSize">
             <size>
              <width>640</width>
              <height>480</height>
             </size>
            </property>
            <property name="text">
             <string>TextLabel</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="verticalSpacer_5">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <spacer name="verticalSpacer_3">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item>
         <spacer name="verticalSpacer_4">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <spacer name="verticalSpacer_6">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_4">
            <item>
             <widget class="QLabel" name="label_2">
              <property name="minimumSize">
               <size>
                <width>40</width>
                <height>50</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>100</width>
                <height>50</height>
               </size>
              </property>
              <property name="font">
               <font>
                <family>Arial Black</family>
                <pointsize>12</pointsize>
                <weight>75</weight>
                <bold>true</bold>
               </font>
              </property>
              <property name="text">
               <string>No</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QTextEdit" name="textEdit_testerNo">
              <property name="minimumSize">
               <size>
                <width>260</width>
                <height>35</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>200</width>
                <height>35</height>
               </size>
              </property>
              <property name="font">
               <font>
                <family>Arial</family>
                <pointsize>10</pointsize>
                <weight>50</weight>
                <bold>false</bold>
                <stylestrategy>PreferAntialias</stylestrategy>
                <kerning>true</kerning>
               </font>
              </property>
              <property name="layoutDirection">
               <enum>Qt::LeftToRight</enum>
              </property>
              <property name="frameShape">
               <enum>QFrame::Box</enum>
              </property>
              <property name="frameShadow">
               <enum>QFrame::Plain</enum>
              </property>
              <property name="html">
               <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Arial'; font-size:10pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-family:'微軟正黑體'; font-size:12pt;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="btnBuildTester">
              <property name="minimumSize">
               <size>
                <width>90</width>
                <height>35</height>
               </size>
              </property>
              <property name="font">
               <font>
                <family>微軟正黑體</family>
                <pointsize>10</pointsize>
                <weight>75</weight>
                <bold>true</bold>
               </font>
              </property>
              <property name="text">
               <string>建立新採樣測試</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="btnCameraStart_r">
              <property name="minimumSize">
               <size>
                <width>90</width>
                <height>35</height>
               </size>
              </property>
              <property name="font">
               <font>
                <family>微軟正黑體</family>
                <pointsize>10</pointsize>
               </font>
              </property>
              <property name="text">
               <string>擷取右眼影像</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="btnCameraStart_l">
              <property name="minimumSize">
               <size>
                <width>90</width>
                <height>35</height>
               </size>
              </property>
              <property name="font">
               <font>
                <family>微軟正黑體</family>
                <pointsize>10</pointsize>
               </font>
              </property>
              <property name="text">
               <string>擷取左眼影像</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer_3">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_3">
            <item>
             <widget class="QLabel" name="label_showInfo">
              <property name="minimumSize">
               <size>
                <width>308</width>
                <height>35</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>308</width>
                <height>35</height>
               </size>
              </property>
              <property name="font">
               <font>
                <family>Arial Black</family>
                <pointsize>10</pointsize>
                <weight>75</weight>
                <bold>true</bold>
               </font>
              </property>
              <property name="midLineWidth">
               <number>0</number>
              </property>
              <property name="text">
               <string/>
              </property>
              <property name="scaledContents">
               <bool>false</bool>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="btn_cancelWriteVideo">
              <property name="minimumSize">
               <size>
                <width>90</width>
                <height>35</height>
               </size>
              </property>
              <property name="font">
               <font>
                <family>微軟正黑體</family>
                <pointsize>10</pointsize>
               </font>
              </property>
              <property name="text">
               <string>取消擷取</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="btn_tester_finish">
              <property name="minimumSize">
               <size>
                <width>90</width>
                <height>35</height>
               </size>
              </property>
              <property name="font">
               <font>
                <family>微軟正黑體</family>
                <pointsize>10</pointsize>
               </font>
              </property>
              <property name="text">
               <string>採樣結束</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="btnOpenFolder">
              <property name="minimumSize">
               <size>
                <width>90</width>
                <height>35</height>
               </size>
              </property>
              <property name="font">
               <font>
                <family>微軟正黑體</family>
                <pointsize>10</pointsize>
               </font>
              </property>
              <property name="text">
               <string>開啟資料夾</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="btnVideoSelUpload">
              <property name="minimumSize">
               <size>
                <width>90</width>
                <height>35</height>
               </size>
              </property>
              <property name="font">
               <font>
                <family>微軟正黑體</family>
                <pointsize>10</pointsize>
               </font>
              </property>
              <property name="text">
               <string>選擇檔案上傳</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer_2">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QPlainTextEdit" name="txtShow">
        <property name="font">
         <font>
          <family>微軟正黑體</family>
          <pointsize>10</pointsize>
         </font>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="3" column="0">
     <spacer name="verticalSpacer_2">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="2" column="0">
     <layout class="QVBoxLayout" name="verticalLayout_5">
      <item>
       <widget class="Line" name="line">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="label_logo">
          <property name="text">
           <string/>
          </property>
          <property name="pixmap">
           <pixmap resource="icon.qrc">:/deepLogo_s.png</pixmap>
          </property>
          <property name="scaledContents">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_version">
          <property name="palette">
           <palette>
            <active>
             <colorrole role="Text">
              <brush brushstyle="SolidPattern">
               <color alpha="255">
                <red>90</red>
                <green>90</green>
                <blue>90</blue>
               </color>
              </brush>
             </colorrole>
            </active>
            <inactive>
             <colorrole role="Text">
              <brush brushstyle="SolidPattern">
               <color alpha="255">
                <red>90</red>
                <green>90</green>
                <blue>90</blue>
               </color>
              </brush>
             </colorrole>
            </inactive>
            <disabled>
             <colorrole role="Text">
              <brush brushstyle="SolidPattern">
               <color alpha="255">
                <red>120</red>
                <green>120</green>
                <blue>120</blue>
               </color>
              </brush>
             </colorrole>
            </disabled>
           </palette>
          </property>
          <property name="font">
           <font>
            <family>Arial</family>
            <pointsize>9</pointsize>
            <weight>75</weight>
            <italic>true</italic>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>V2024.01.03.7</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btnCameraStart">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="font">
           <font>
            <family>微軟正黑體</family>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="text">
           <string>開始擷取影像</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btnCameraStop">
          <property name="font">
           <font>
            <family>微軟正黑體</family>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="text">
           <string>camera stop</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btnVideoSeltoNolight">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="font">
           <font>
            <family>微軟正黑體</family>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="text">
           <string>選取檔案去除亮點</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btnVideoSetting">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="font">
           <font>
            <family>微軟正黑體</family>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="text">
           <string>讀取相機資訊</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btnVideoSeltoCanny">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="font">
           <font>
            <family>微軟正黑體</family>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="text">
           <string>選取檔案偵測邊緣</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btnROINUpload">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="font">
           <font>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="text">
           <string>ROI提取並上傳</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1136</width>
     <height>28</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微軟正黑體</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <widget class="QMenu" name="menu">
    <property name="font">
     <font>
      <family>微軟正黑體</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="title">
     <string>上傳設定</string>
    </property>
    <addaction name="action_upload_set"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="font">
     <font>
      <family>微軟正黑體</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="title">
     <string>影片格式</string>
    </property>
    <addaction name="actio_video_set"/>
   </widget>
   <widget class="QMenu" name="menu_3">
    <property name="font">
     <font>
      <family>微軟正黑體</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="title">
     <string>說明</string>
    </property>
    <addaction name="action_about"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
   <addaction name="menu_3"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="font">
    <font>
     <family>微軟正黑體</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="action30s">
   <property name="text">
    <string>30s</string>
   </property>
  </action>
  <action name="action_about">
   <property name="icon">
    <iconset resource="icon.qrc">
     <normaloff>:/deepicon.png</normaloff>:/deepicon.png</iconset>
   </property>
   <property name="text">
    <string> 關於eyeAnalysis</string>
   </property>
  </action>
  <action name="action10s_2">
   <property name="text">
    <string>10s</string>
   </property>
  </action>
  <action name="action20s_2">
   <property name="text">
    <string>20s</string>
   </property>
  </action>
  <action name="action30s_2">
   <property name="text">
    <string>30s</string>
   </property>
  </action>
  <action name="action_default">
   <property name="text">
    <string>預設</string>
   </property>
  </action>
  <action name="action_set">
   <property name="text">
    <string>新增</string>
   </property>
   <property name="iconText">
    <string>修改</string>
   </property>
   <property name="toolTip">
    <string>修改</string>
   </property>
  </action>
  <action name="action_upload_set">
   <property name="text">
    <string>設定</string>
   </property>
  </action>
  <action name="actio_video_set">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>設定</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="icon.qrc"/>
 </resources>
 <connections/>
</ui>
